SetupDataPkg/Tools/VariableList.py

```diff
@@ -1178,24 +1178,29 @@ def read_csv(schema, csv_path):
             knob.value = knob.format.string_to_object(knob_value_string)
 
 
-def write_csv(schema, csv_path, subknobs=True):
+def write_csv(schema, csv_path, full, subknobs=True):

```

Comment:
Looks like the calls to `write_csv()` on Line 1276 and Line 1289 need to be updated?

Reply:
Good catch, thanks. Updated.